CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @getsentry/owners-snuba
+/rust-arroyo @getsentry/owners-snuba @getsentry/owners-processing
```

Comment:
apparently this team no longer exists?

Reply:
```suggestion
/rust-arroyo @getsentry/owners-snuba @getsentry/processing
```

it appears it has been wrong in snuba the entire time